variables.tf

```diff
@@ -104,6 +104,19 @@ variable "engine_version" {
   # http://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html
 }
 
+variable "major_engine_version" {
+  type        = "string"
+  description = "Database MAJOR engine version, depends on engine type"
+
+  # https://docs.aws.amazon.com/cli/latest/reference/rds/create-option-group.html
```

Comment:
I think we need to add `default = ""` here since it will always be required even if `var.option_group_name` is provided by the user

Reply:
Ok, I'll revert that and commit